WordPress/Classes/ViewRelated/Site Creation/WebAddress/WebAddressTableViewProvider.swift

```diff
@@ -45,6 +45,10 @@ final class WebAddressTableViewProvider: NSObject, TableViewProvider {
         return data.count
     }
 
+    func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        return NSLocalizedString("Suggestions", comment: "Suggested domains")
```

Comment:
We've had some feedback about the appearance of the table view style in `p1550806679052200-slack-ios-osx-dev` that I hope you find useful.

Reply:
Thanks so much, @stevebaranski! Applied the changes you suggested and it worked out perfect 👌